### PR TITLE
fix: Remove dependency on github.com/pkg/errors

### LINKED ISSUE #113627: Remove dependency on github.com/pkg/errors
**Why is this needed?**

The functionality that the package provides is already present in golang's builtin errors (using `%w` in `fmt.Errof`) and the github repo is now read-only.

/kind cleanup
/priority backlog
/good-first-issue

**Notes to people that want to collaborate**

Avoid creating a single PR for the entire repo. Create multiple PRs, each targeting a specific SIG.

Do **not** submit PRs for `cmd/kubeadm`

### PR DESCRIPTION
**Pull request by falser101**
Signed-off-by: jianfei.zhang <jianfei.zhang@daocloud.io>

#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes:
Fixes #113627

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

**Comment by falser101** [comment]
@pacoxu

File: cmd/kubelet/app/init_windows.go [hunk 1]
```diff
@@ -57,7 +56,7 @@ func getPriorityValue(priorityClassName string) uint32 {
 func createWindowsJobObject(pc uint32) (windows.Handle, error) {
 	job, err := windows.CreateJobObject(nil, nil)
 	if err != nil {
-		return windows.InvalidHandle, errors.Wrap(err, "windows.CreateJobObject failed")
+		return windows.InvalidHandle, fmt.Errorf("windows.CreateJobObject failed, err: %w", err)
```
Comment (klueska) [inline cmd/kubelet/app/init_windows.go:59]: I would remove the `, err` at the end of each of these error strings.

**Comment by klueska** [comment]
I will approve this now (because it's late for me and I won't be around to approve it later), but please address my comment above before giving an LGTM

/approve

File: cmd/kubelet/app/init_windows.go (hunk 1 above)
Comment (falser101) [reply cmd/kubelet/app/init_windows.go]: down

**Comment by falser101** [comment]
> I will approve this now (because it's late for me and I won't be around to approve it later), but please address my comment above before unholding it
> 
> /approve /hold

down!

**Review by pacoxu — COMMENTED** [review]
/unhold
/lgtm

**Comment by falser101** [comment]
This was approved before code freeze. Can anyone help to add the milestone label?

https://github.com/orgs/kubernetes/teams/milestone-maintainers

**Comment by dims** [comment]
this is not a bug or a test problem, let's land it when 1.27 opens up

/milestone v1.27

**Pull request merged**